src/main/java/org/web3j/quorum/JsonRpc2_0Quorum.java

```diff
@@ -76,4 +90,324 @@ public JsonRpc2_0Quorum(
                 web3jService,
                 PrivatePayload.class);
     }
+
+    @Override
+    public Request<?, EthSendTransaction> ethSendTransactionAsync(PrivateTransaction transaction) {
+        return new Request<>(
+                "eth_sendTransactionAsync",
+                Collections.singletonList(transaction),
+                web3jService,
+                EthSendTransaction.class);
+    }
+
+    // raft consensus
+
+    @Override
+    public Request<?, RaftLeader> raftGetLeader() {
+        return new Request<>(
+                "raft_leader", Collections.emptyList(), web3jService, RaftLeader.class);
+    }
+
+    @Override
+    public Request<?, RaftRole> raftGetRole() {
+        return new Request<>("raft_role", Collections.emptyList(), web3jService, RaftRole.class);
+    }
+
+    @Override
+    public Request<?, RaftPeerId> raftAddPeer(String enode) {
+        return new Request<>(
+                "raft_addPeer", Collections.singletonList(enode), web3jService, RaftPeerId.class);
+    }
+
+    @Override
+    public Request<?, ConsensusNoResponse> raftRemovePeer(int peerId) {
+        return new Request<>(
+                "raft_removePeer",
+                Collections.singletonList(peerId),
+                web3jService,
+                ConsensusNoResponse.class);
+    }
+
+    @Override
+    public Request<?, RaftCluster> raftGetCluster() {
+        return new Request<>(
+                "raft_cluster", Collections.emptyList(), web3jService, RaftCluster.class);
+    }
+
+    // istanbul consensus
```

Comment:
Might be best to add implementation for `istanbulNodeAddress`, `getSignersFromBlockByHash /Number`.

Reply:
these have been added